### PR TITLE
test: fix fit/fdescribe

### PR DESCRIPTION
**Pull request by alexeagle**
Pick up the fix from @bazel/jasmine package

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,7 +36,7 @@
     "@angular-devkit/core": "^7.3.2",
     "@angular-devkit/schematics": "^7.3.2",
     "@angular/bazel": "file:./tools/npm/@angular_bazel",
-    "@bazel/jasmine": "0.27.4",
+    "@bazel/jasmine": "~0.27.5",
```
Comment (filipesilva) [inline package.json]: Is the unpinned dep intentional? The others like it are pinned.

**Review by filipesilva — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (alexeagle) [reply package.json]: not intentional but also not important I think

**Comment by alexeagle** [comment]
assist: saucelabs is flaking, and circleci is busted

**Comment by alexeagle** [comment]
wait, don't merge
looks like the failure could be real

**Comment by alexeagle** [comment]
kinda wedged until circleci comes back online

**Review by matsko — APPROVED** [review]

**Comment by alexeagle** [comment]
I'll let @gregmagolan take this over, he already has a PR to upgrade that doesn't involve changing Jasmine version used outside bazel

**Pull request closed without merging**